mobile/src/main/res/layout/connect_manually_verification.xml

```diff
@@ -98,6 +98,7 @@
         android:layout_marginHorizontal="@dimen/activity_horizontal_margin"
         android:background="@drawable/bg_round_orange16_btn"
         app:text_color="@color/btn_dark_letter"
+        app:text_all_caps="true"
```

Comment:
hey girls, random random comment. I was chacking the code and saw this commit, I have question, nothing urgent. Don't we have a general style for buttons? It would be nice to refactor all the buttons in Tella. 

Is there a way to -instead of setting text_al_caps for these 6 buttons one by one- we could apply it to all tella buttons, is this possible? on your documentation of reusable component it looks like we have one, right? https://horizontal-org.slite.com/app/docs/rWRlgWNNHuCqWM#388cb339

Reply:
@ahlem-jarrar @wafalakhal  what do you think about this idea? would it be possible to this in order to not need to remember set text_all_caps to all buttons?